Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -1040,59 +1039,127 @@ def create_assets_report_command():
     assets = str(demisto.args()['assets']).split(',')
     template = demisto.args().get('template')
     name = demisto.args().get('name', 'report ' + str(datetime.now()))
-    report_format = demisto.args().get('format', 'pdf')
+    report_format = demisto.args().get('format') or 'pdf'
```

Comment:
why did you change it?

Reply:
When executing it through the playbook the 'format' key exists so the default value doesn't get picked.